wallet/src/server/protos/schema.proto

```diff
@@ -12,8 +12,8 @@ service Wallet {
   rpc GetWalletBalance (GetWalletBalanceRequest) returns (GetWalletBalanceReply) {}
   rpc SetWalletCurrency (SetWalletCurrencyRequest) returns (SetWalletCurrencyReply) {}
   rpc SetWalletCountry (SetWalletCountryRequest) returns (SetWalletCountryReply) {}
-  rpc GetWalletEstablishedCurrency (GetWalletEstablishedCurrencyRequest) returns (GetWalletEstablishedCurrencyReply) {}
-  rpc GetWalletEstablishedCountry (GetWalletEstablishedCountryRequest) returns (GetWalletEstablishedCountryReply) {}
+  rpc GetWalletCurrency (GetWalletCurrencyRequest) returns (GetWalletCurrencyReply) {}
+  rpc GetWalletCountry (GetWalletCountryRequest) returns (GetWalletCountryReply) {}
```

Comment:
@Ktoxcon 
I removed "Established". It is not a common word to use in these cases. Only `GetWalletCurrencyCode` and `GetWalletCountryCode` are enough. 

If we were to use a term like "Established"  in the future, it is better to use: "Set". For example: `GetWalletSetCurrencyCode`, but it is unnecessary.